src/domain/validate-yesnopartial/repositories/NHWAYesNoPartialDataValuesRepository.ts

```diff
@@ -0,0 +1,17 @@
+import { Config } from "../../common/entities/Config";
+import { PaginatedObjects, Paging, Sorting } from "../../common/entities/PaginatedObjects";
+import { DataValueItem, DataValueItemIdentifier } from "../entities/DataValueItem";
+import { Id } from "../../common/entities/Base";
+
+export interface NHWAYesNoPartialDataValuesRepository {
+    get(options: NHWAYesNoPartialDataValuesRepositoryGetOptions): Promise<PaginatedObjects<DataValueItem>>;
```

Comment:
[note] On Clean arch, typically we have an entity+repo that work together. Here we have NHWAYesNoPartialDataValues/DataValueItem.